src/analytics/formTracker/formTracker.test.ts

```diff
@@ -22,64 +29,56 @@ describe("FormTracker", () => {
   });
 
   test("getFieldValue should return field value", () => {
-    const instance = new FormTracker();
     const fields: FormField[] = [
       { id: "test", name: "test", value: "test value", type: "test" },
     ];
     expect(instance.getFieldValue(fields)).toBe("test value");
   });
 
   test("getFieldType should return free text field if type is text", () => {
-    const instance = new FormTracker();
     const fields: FormField[] = [
       { id: "test", name: "test", value: "test value", type: "text" },
     ];
     expect(instance.getFieldType(fields)).toBe(instance.FREE_TEXT_FIELD_TYPE);
   });
 
   test("getFieldType should return free text field if type is textarea", () => {
-    const instance = new FormTracker();
     const fields: FormField[] = [
       { id: "test", name: "test", value: "test value", type: "textarea" },
     ];
     expect(instance.getFieldType(fields)).toBe(instance.FREE_TEXT_FIELD_TYPE);
   });
 
   test("getFieldType should return drop-down list if type is select-one", () => {
-    const instance = new FormTracker();
     const fields: FormField[] = [
       { id: "test", name: "test", value: "test value", type: "select-one" },
     ];
     expect(instance.getFieldType(fields)).toBe("drop-down list");
   });
 
   test("getFieldType should return checkbox if type is checkbox", () => {
-    const instance = new FormTracker();
     const fields: FormField[] = [
       { id: "test", name: "test", value: "test value", type: "checkbox" },
     ];
     expect(instance.getFieldType(fields)).toBe("checkbox");
   });
 
   test("getFieldType should return radio buttons if type is radio", () => {
-    const instance = new FormTracker();
     const fields: FormField[] = [
       { id: "test", name: "test", value: "test value", type: "radio" },
     ];
     expect(instance.getFieldType(fields)).toBe("radio buttons");
   });
 
   test("getFieldLabel should return field label", () => {
-    const instance = new FormTracker();
     const label = document.createElement("label");
-    label.innerHTML = "test label";
     label.textContent = "test label";
     document.body.appendChild(label);
     expect(instance.getFieldLabel()).toBe("test label");
+    document.body.removeChild(label);
```

Comment:
Maybe this line is not useful anymore, with the beforeeach function.

Reply:
removed